MiAZ/miaz.py

```
@@ -1,111 +1,59 @@
 #!@PYTHON@
 
-# Copyright 2024 Tomás Vírseda
+# Copyright 2019-2025 Tomás Vírseda
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import os
 import sys
 import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)